scripts/deployment/phase2-assets/collaterals/deploy_convex_stable_plugin.ts

```diff
@@ -0,0 +1,115 @@
+import fs from 'fs'
+import hre, { ethers } from 'hardhat'
+import { getChainId } from '../../../../common/blockchain-utils'
+import { networkConfig } from '../../../../common/configuration'
+import { bn } from '../../../../common/numbers'
+import {
+  getDeploymentFile,
+  getAssetCollDeploymentFilename,
+  IAssetCollDeployments,
+  getDeploymentFilename,
+  fileExists,
+} from '../../common'
+import { CvxStableCollateral } from '../../../../typechain'
+import {
+  CurvePoolType,
+  DAI_ORACLE_ERROR,
+  DAI_ORACLE_TIMEOUT,
+  DAI_USD_FEED,
+  DEFAULT_THRESHOLD,
+  DELAY_UNTIL_DEFAULT,
+  MAX_TRADE_VOL,
+  PRICE_TIMEOUT,
+  THREE_POOL,
+  THREE_POOL_TOKEN,
+  USDC_ORACLE_ERROR,
+  USDC_ORACLE_TIMEOUT,
+  USDC_USD_FEED,
+  USDT_ORACLE_ERROR,
+  USDT_ORACLE_TIMEOUT,
+  USDT_USD_FEED,
+} from '../../../../test/plugins/individual-collateral/convex/constants'
+
+// This file specifically deploys Convex Stable Plugin for 3pool
+
+async function main() {
+  // ==== Read Configuration ====
+  const [deployer] = await hre.ethers.getSigners()
+
+  const chainId = await getChainId(hre)
+
+  console.log(`Deploying Collateral to network ${hre.network.name} (${chainId})
+    with burner account: ${deployer.address}`)
+
+  if (!networkConfig[chainId]) {
+    throw new Error(`Missing network configuration for ${hre.network.name}`)
+  }
+
+  // Get phase1 deployment
+  const phase1File = getDeploymentFilename(chainId)
+  if (!fileExists(phase1File)) {
+    throw new Error(`${phase1File} doesn't exist yet. Run phase 1`)
+  }
+  // Check previous step completed
+  const assetCollDeploymentFilename = getAssetCollDeploymentFilename(chainId)
+  const assetCollDeployments = <IAssetCollDeployments>getDeploymentFile(assetCollDeploymentFilename)
+
+  const deployedCollateral: string[] = []
+
+  /********  Deploy Convex Stable Pool for 3pool  **************************/
+
+  const CvxMiningFactory = await ethers.getContractFactory('CvxMining')
+  const CvxMining = await CvxMiningFactory.deploy()
+
+  const CvxStableCollateralFactory = await hre.ethers.getContractFactory('CvxStableCollateral')
+  const ConvexStakingWrapperFactory = await ethers.getContractFactory('ConvexStakingWrapper', {
+    libraries: { CvxMining: CvxMining.address },
+  })
+
+  const w3Pool = await ConvexStakingWrapperFactory.deploy()
+  await w3Pool.initialize(9)
+
+  const collateral = <CvxStableCollateral>await CvxStableCollateralFactory.connect(deployer).deploy(
+    {
+      erc20: w3Pool.address,
+      targetName: ethers.utils.formatBytes32String('USD'),
+      priceTimeout: PRICE_TIMEOUT,
+      chainlinkFeed: DAI_USD_FEED, // unused but cannot be zero
+      oracleError: bn('1'), // unused but cannot be zero
+      oracleTimeout: bn('1'), // unused but cannot be zero
+      maxTradeVolume: MAX_TRADE_VOL,
+      defaultThreshold: DEFAULT_THRESHOLD,
+      delayUntilDefault: DELAY_UNTIL_DEFAULT,
+    },
+    bn('1e14'), // revenueHiding = 0.01%
```

Comment:
We should either remove revenueHiding entirely or set it to the value in `scripts/deployment/utils.ts` (fp('1e-6')), given that we think the virtual price cannot decrease.